fe/fe-core/src/main/java/com/starrocks/connector/hudi/HudiConnector.java

```diff
@@ -77,12 +74,6 @@ private HudiMetadataFactory createMetadataFactory() {
     }
 
     public void onCreate() {
-        if (!CatalogMgr.ResourceMappingCatalog.isResourceMappingCatalog(catalogName) &&
```

Comment:
why not support for hudi

Reply:
Because can not can partition modified time for now